### PR TITLE
feat: Added buttons to library.

### PR DESCRIPTION
**Pull request by abhijeetekad**
Buttons types:
1) Primary button.
2)Outline button.
3)Floating button.
4)Link button
CPT2202031406-1731x878
.

### CODE REVIEW

File: documentation/doc.html [hunk 1]
```diff
@@ -265,7 +265,74 @@ <h3>Horizontal Card</h3>
...
+  style="width: 750px; height: 189px; border:0; transform: scale(1); overflow:hidden;"
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
```
Comment (amarphule) [inline documentation/doc.html]: change code snippet accordingly. here you used Outline buttons code snippet

File: documentation/doc.html [hunk 2]
```diff
@@ -265,7 +265,74 @@ <h3>Horizontal Card</h3>
...
```
Comment (amarphule) [inline documentation/doc.html]: change code snippet accordingly. here you used Outline buttons code snippet

**Review by amarphule — CHANGES_REQUESTED** [review]
The code and UI look Great. Change code snippet accordingly. You used the Outline buttons code snippet everywhere.

**Review by nisargpawade — CHANGES_REQUESTED** [review]
please change the description below simple button heading as the class name you used are different

File: documentation/doc.html (hunk 1 above)
Comment (abhijeetekad) [reply documentation/doc.html]: Done..

File: documentation/doc.html (hunk 2 above)
Comment (abhijeetekad) [reply documentation/doc.html]: Done

**Review by nisargpawade — APPROVED** [review]

**Review by nisargpawade — APPROVED** [review]

**Pull request merged**